examples/svg-components/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "svg-components",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "next": "beta"
+  },
+  "devDependencies": {
+    "babel-plugin-inline-react-svg": "^0.2.0"
```

Comment:
Oh hey, looks like there's a babel plugin there too. My observation skills may not be the best. I'll give it a shot soon =P